core/src/main/java/org/openmbee/mms/core/objects/ElementsCommitResponse.java

```diff
@@ -0,0 +1,13 @@
+package org.openmbee.mms.core.objects;
+
+public class ElementsCommitResponse extends ElementsResponse {
+    private String commitId;
```

Comment:
add @Schema(nullable = true) for openapi

Reply:
added